### ISSUE TITLE
Don't depend on a #notebook element existing

### ISSUE THREAD

**Issue by jasongrout** [bloomberg/bqplot#382]
In the two show_tooltip methods:

https://github.com/bloomberg/bqplot/blob/3dd09694fce17841787bcb0a26b21079d62ee167/js/src/MarketMap.js#L604

https://github.com/bloomberg/bqplot/blob/e3501c367af3f91559d702896ec49ba73189a614/js/src/Mark.js#L250

there are references to the `#notebook` document element. This may not exist (embedded plots, or in JLab, etc.). We should not depend on DOM structure like this.

**Comment by dmadeka** [comment]
@jasongrout do you think we could use `this.el` there instead?

**Comment by ssunkara1** [comment]
As of the last time I checked, I could not find any other way of getting the tooltip positioning to be correct when we scroll (horizontally (there are two types of this. 1. When the entire browser window is scrollable. 2. When only the widget output area is scrollable)  and vertically)/full screen etc. Depending on the `#notebook` element was a temporary fix to the problem as I discussed with @jasongrout at the time I was making that change. We do need to figure out a better way of doing it.
